src/index.html

```diff
@@ -15,8 +15,99 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+    />
```

Comment:
There is an extra closing tag (`/>`) on this line which is not needed. It should be removed to prevent potential rendering issues.